README.md

```diff
@@ -8,33 +8,52 @@ Prerequisites
 The download-script requires the `lxml` package.
 You can install it via `pip install -r requirements.txt`.
 
-Setup
------
+Install
+-------
 
 ```
-cp settings.sample.py settings.py
+pip install wfs-downloader
 ```
 
-Edit `settings.py` for services.
-
-
 Usage
 -----
 
+Create a `config.yml` specifying your setup like this:
+
+```yml
+url: http://fbinter.stadt-berlin.de/fb/wfs/data/senstadt/s_wfs_baumbestand_an
+layer: fis:s_wfs_baumbestand_an
+
+bbox:
+  west:   370000.0
+  south: 5800000.0
+  east:   415000.0
+  north: 5837000.0
+
+size: 10000
```

Comment:
It might be worth explaining somewhere what this setting does but other than that looks good! 👍